src/developers/notary_server.md

```diff
@@ -55,7 +55,7 @@ docker run --init -p 127.0.0.1:7047:7047 ghcr.io/tlsnotary/tlsn/notary-server:<v
 ```
 
 ## API Endpoints
-Please refer to the list of all API endpoints [here](https://editor.swagger.io/?url=https://raw.githubusercontent.com/tlsnotary/tlsn/main/notary-server/openapi.yaml#/).
+Please refer to the list of all API endpoints [here](./swagger.html).
```

Comment:
need to change this path too right?

Reply:
Right, missed that. My bad. Fixed and pushed 😅

Reply:
no probs haha